pkg/shell/shell.css

```diff
@@ -627,6 +627,16 @@ ul.available-interfaces-group {
     vertical-align: top;
 }
 
+/* workaround: make text red for select if contained in has-error, not in patternfly */
```

Comment:
> Has this been filed up stream as an issue? Or does it stem from our use of an old patternfly?

@andreasn wanted to look at it and file an issue if necessary. He recommend the proposed approach (see comments above).